propagator/ottrace/lib/opentelemetry/propagator/ottrace.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'opentelemetry-api'
+require 'opentelemetry/propagator/ottrace/version'
+
+module OpenTelemetry
+  module Propagator
+    module OTTrace
+      extend self
+
+      TRACE_ID_HEADER = 'ot-tracer-traceid'
+      SPAN_ID_HEADER = 'ot-tracer-spanid'
+      SAMPLED_HEADER = 'ot-tracer-sampled'
+      BAGGAGE_HEADER_PREFIX = 'ot-baggage-'
+
+      # TODO: Is this documented in a specification somewhere? Does check need to happen in all injectors?
```

Comment:
I believe it's specified here https://github.com/open-telemetry/opentelemetry-specification/blob/14d123c121b6caa53bffd011292c42a181c9ca26/specification/context/api-propagators.md#textmap-propagator , so I think we're on spec

We may want to extract this into `/common` at some point in the future if we have to form http header keys dynamically for other propagation methods, i think it's fine living here for now

Reply:
I will make sure to reference those docs in the comment.